modin/pandas/test/test_expanding.py

```diff
@@ -0,0 +1,129 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import pytest
+import numpy as np
+import pandas
+import modin.pandas as pd
+
+from .utils import (
+    df_equals,
+    test_data_values,
+    test_data_keys,
+    create_test_dfs,
+    default_to_pandas_ignore_string,
+)
+from modin.config import NPartitions
+
+NPartitions.put(4)
+
+
+def create_test_series(vals):
+    if isinstance(vals, dict):
+        modin_series = pd.Series(vals[next(iter(vals.keys()))])
+        pandas_series = pandas.Series(vals[next(iter(vals.keys()))])
+    else:
+        modin_series = pd.Series(vals)
+        pandas_series = pandas.Series(vals)
+    return modin_series, pandas_series
+
+
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+@pytest.mark.parametrize("min_periods", [None, 5])
+@pytest.mark.parametrize("axis", [0, 1])
+def test_dataframe(data, min_periods, axis):
+    modin_df = pd.DataFrame(data)
+    pandas_df = pandas.DataFrame(data)
+    pandas_expanded = pandas_df.expanding(
+        min_periods=min_periods,
+        center=True,
+        axis=axis,
+    )
+    modin_expanded = modin_df.expanding(
+        min_periods=min_periods,
+        center=True,
+        axis=axis,
+    )
+
+    df_equals(modin_expanded.count(), pandas_expanded.count())
+    df_equals(modin_expanded.sum(), pandas_expanded.sum())
+    df_equals(modin_expanded.mean(), pandas_expanded.mean())
+    df_equals(modin_expanded.var(ddof=0), pandas_expanded.var(ddof=0))
+    df_equals(modin_expanded.std(ddof=0), pandas_expanded.std(ddof=0))
+    df_equals(modin_expanded.min(), pandas_expanded.min())
+    df_equals(modin_expanded.max(), pandas_expanded.max())
+
+
+@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
+@pytest.mark.parametrize("min_periods", [None, 5])
+def test_dataframe_agg(data, min_periods):
```

Comment:
I prefer that we parametrize this and the following test cases as well.

Reply:
I'll parametrize the series methods test. I think it's cleaner to keep the `agg` test separate since I ran into some failures when I was initially porting the method, and `agg` also doesn't allow `axis=1`.